minos/api_gateway/rest/handler.py

```diff
@@ -28,10 +38,29 @@ async def orchestrate(request: web.Request) -> web.Response:
 
     discovery_data = await discover(discovery_host, int(discovery_port), "/microservices", verb, url)
 
-    microservice_response = await call(**discovery_data, original_req=request)
+    user = await get_user(request)
+
+    microservice_response = await call(**discovery_data, original_req=request, user=user)
     return microservice_response
 
 
+async def get_user(request) -> str:
+    try:
+        await get_token(request)
+    except NoTokenException:
+        return ANONYMOUS
+    else:
```

Comment:
(A coding style hint) I think is preferable to stop using `else` with `try-except` statements when a `return` is performed on the `except` section because it adds an additional unnecessary indentation level.

Reply:
As we've discussed by chat I'm leaving this so, since it's clearer it's related to the same try/except structure.